include/bluetooth/mesh/cfg_cli.h

```diff
@@ -936,6 +940,97 @@ int32_t bt_mesh_cfg_cli_timeout_get(void);
  */
 void bt_mesh_cfg_cli_timeout_set(int32_t timeout);
 
+/** Parsed Composition data page 0 representation.
+ *
+ *  Should be pulled from the return buffer passed to
+ *  @ref bt_mesh_cfg_comp_data_get using
+ *  @ref bt_mesh_comp_page_0_get.
+ */
+struct bt_mesh_comp_page_0 {
+	/** Company ID */
+	uint16_t cid;
+	/** Product ID */
+	uint16_t pid;
+	/** Version ID */
+	uint16_t vid;
+	/** Replay protection list size */
+	uint16_t crpl;
+	/** Supported features, see @ref BT_MESH_FEAT_SUPPORTED. */
+	uint16_t feat;
+
+	struct net_buf_simple *_buf;
+};
+
+/** Composition data page 0 element representation */
+struct bt_mesh_comp_elem {
```

Comment:
Would `bt_mesh_comp_page_0_elem` be more clear?

Reply:
Probably, yes. I renamed both the composition data page and the elem to use a `bt_mesh_comp_p0` prefix.